### PR TITLE
Task 3.2

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: Practicum/Practicum.csproj [hunk 1]
```diff
@@ -79,7 +79,9 @@
     <Content Include="JSBook\ex2.css" />
     <Content Include="JSBook\ex2.html" />
     <Content Include="JSBook\ex2.js" />
-
+    <Content Include="JSBook\Z3.2.css" />
```
Comment (KasichMax) [inline Practicum/Practicum.csproj]: Redundant line - you don't have such file.

**Comment by akravchenko564** [comment]
Done 97ef44d9c0913a11f60687626d0e3d949e009ec3

File: Practicum/Practicum.csproj (hunk 1 above)
Comment (akravchenko564) [inline Practicum/Practicum.csproj]: Done b9cd1013b1745628a1b2e10328f813f16dda0aa7

File: Practicum/JSBook/Z3.2.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+	<title>Z3</title>
```
Comment (KasichMax) [inline Practicum/JSBook/Z3.2.html]: Why Z3? MB Z3.2?

File: Practicum/JSBook/Z3.2.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿function checkSpam(str) {
+	var lowerCase = str.toLowerCase();
+	if (~lowerCase.indexOf("xxx") || ~lowerCase.indexOf("yyy")) {
+		return true;
+	}
+	else {
+		return false;
+	}
```
Comment (KasichMax) [inline Practicum/JSBook/Z3.2.js]: `~lowerCase.indexOf("xxx") || ~lowerCase.indexOf("yyy")` returns bool result.
You don't need to use `if(...) return true else return false;`
You can just write `return ~lowerCase.indexOf("xxx") || ~lowerCase.indexOf("yyy")`

Comment (akravchenko564) [inline Practicum/JSBook/Z3.2.js]: This `return ~lowerCase.indexOf("xxx") || ~lowerCase.indexOf("yyy")` returns int, not bool.

File: Practicum/JSBook/Z3.2.html (hunk 2 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z3.2.html]: Done 866ad5ad7bd1f13ad83f4cb8ad720953bdde5fd1

File: Practicum/JSBook/Z3.2.js (hunk 3 above)
Comment (KasichMax) [inline Practicum/JSBook/Z3.2.js]: You're right.

**Pull request merged**